### PR TITLE
Reorder sidebar & add sshkey, orchestrators, vms pages

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Changes

 - reordered and renamed items in side bar to reflect order in attached issue
 - created sshkey page
 - removed sshkey from profile manager
 - created vms page that contains Full and Micro vms cards
 - created orchestrators page that contains kubernetes and caprover cards

### Related Issues

#1861
image

Screenshot from 2024-01-04 13-52-27

Screenshot from 2024-01-04 13-52-41

image

### CODE REVIEW

**Comment by Mahmoud-Emad** [comment]
Please take care of the logos in the light mode 
image

did we move the machines to an isolated page?
if so, shall we remove them from the applications page or we keep them there?

Please take a look on the Kubernetes and Caprover too

**Comment by Mahmoud-Emad** [comment]
did we move the machines to an isolated page?
if so, shall we remove them from the applications page or we keep them there?

Please take a look on the Kubernetes and Caprover too

File: packages/playground/src/views/orchestrators_view.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<template>
+  <view-layout>
+    <v-row>
+      <v-col v-for="card in cards" :key="card.title">
+        <router-link :to="'/solutions' + card.route">
```
Comment (Mahmoud-Emad) [inline packages/playground/src/views/orchestrators_view.vue:5]: Do we need to rename it to applications?

**Review by Mahmoud-Emad — APPROVED** [review]
Good job :)

**Review by Mahmoud-Emad — APPROVED** [review]

**Review by AlaaElattar — APPROVED** [review]

**Pull request merged**